src/marshmallow/class_registry.py

```diff
@@ -11,16 +11,19 @@
 from marshmallow.exceptions import RegistryError
 
 if typing.TYPE_CHECKING:
-    from marshmallow import Schema
+    # noinspection PyUnresolvedReferences
```

Comment:
What is this for? Can we live without it? Not a big deal, but I'd rather not add pragmas for tools we're not using.

Reply:
This is for PyCharm (a JetBrains python IDE). Otherwise it markes this line as unised an recommends to delete

Reply:
Is this a check specific to PyCharm or is PyCharm using another tool underneath (pylint)?

Reply:
Seems like it's PyCharm's specific tool: https://www.jetbrains.com/help/pycharm/disabling-and-enabling-inspections.html#comments-ref

Here's a question about connecting it with pylint: https://stackoverflow.com/questions/41979763/suppress-warning-for-both-pycharm-and-pylint
So I think that's specific